app/Http/Controllers/FrontController.php

```diff
@@ -83,18 +81,21 @@ public function home() {
         })->sortBy(function ($e) {
             return strtotime($e->date);
         });
+        foreach ($events as $e) {
+            $this->reduceEventBanner($e);
```

Comment:
Same thing as above. I could see how this could get slow. Probably not as huge of an issue here, but I would still rather see this run daily for all events.

Reply:
Done. reduceEventBanner() call removed from FrontController